src/phoenix/datasets/experiments.py

```diff
@@ -66,9 +67,12 @@ def run_experiment(
     experiment_description: Optional[str] = None,
     experiment_metadata: Optional[Mapping[str, Any]] = None,
     repetitions: int = 1,
+    evaluators: Optional[Union[ExperimentEvaluator, Iterable[ExperimentEvaluator]]] = None,
     rate_limit_errors: Optional[Union[Type[BaseException], Tuple[Type[BaseException], ...]]] = None,
 ) -> Experiment:
     assert repetitions > 0, "Must run the experiment at least once."
+    if isinstance(evaluators, (CanEvaluate, CanAsyncEvaluate)):
```

Comment:
```suggestion
    if not isinstance(evaluators, list):
```

The user might implement a type that does not satisfy the duck type. This will help them get a little farther so they can get a more informative error message.